app/Handlers/Commands/Metric/AddMetricPointCommandHandler.php

```diff
@@ -36,7 +36,7 @@ public function handle(AddMetricPointCommand $command)
         ];
 
         if ($createdAt) {
-            $data['created_at'] = Carbon::createFromFormat('U', $createdAt)->format('Y-m-d H:i:s');
+            $data['created_at'] = Carbon::createFromFormat('U', $createdAt, config('cachet.timezone'))->format('Y-m-d H:i:s');
```

Comment:
that stuff is ok in controllers, but i always feel handers need to be framework agnostic, especially when we come to actually test the handlers